Source/MQTTnet/Server/MqttClientSession.cs

```diff
@@ -29,6 +29,8 @@ public class MqttClientSession : IMqttClientSession
         private MqttApplicationMessage _willMessage;
         private bool _wasCleanDisconnect;
         private IMqttChannelAdapter _adapter;
+        private Task<bool> _run;
```

Comment:
In my opinion this should be renamed to _worker_ or _workerTask_ or whatever because _run_ is confusing (to me) 😄

Reply:
will do